packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -850,8 +851,8 @@ function createPositionReference(
 
 export function createSequenceInterval(
 	label: string,
-	start: number,
-	end: number,
+	start: number | LocalReferencePosition,
```

Comment:
these should stay as number-only. You can invoke APIs to make sequence intervals directly from LocalReferencePositions in the sequence interval indexes rather than use the helpers.